### PR TITLE
[TC-6131] [TC-7045] Add polling pattern

### PR DESCRIPTION
**Pull request by marcmatt**
## Description

https://tradecloud.atlassian.net/browse/TC-6131 story
https://tradecloud.atlassian.net/browse/TC-7045 sub task

### Scope
This PR adds
- the polling pattern
- how to choose either webhook POST/PUT versus GET versus polling

https://app.gitbook.com/@tradecloud/s/api/v/TC-6131-add-polling-pattern/

### Submitter pre-flight check

- [x] Review your documentation
- [x] Add labels `needs reviewing`
- [x] Assign PR and JIRA ticket to 'Reviewer' (in 'Reviewing' state)

### Reviewer pre-flight check

- [ ] Review documentation in PR
- [ ] Remove label `needs reviewing`
- [ ] Merge the PR and remove the release for this branch from Gitbook
- [ ] Set Jira ticket to 'Released'

### CODE REVIEW

**Review by Sjorstradecloud — APPROVED** [review]
Gitbook page looks clear, nicely done.

File: buyer/receive/receive-order-response-api.md [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+{% endapi-method-parameter %}
+
+{% api-method-parameter name="Content-Type" type="string" required=true %}
+application/json
+{% endapi-method-parameter %}
+{% endapi-method-headers %}
+
+{% api-method-body-parameters %}
+{% api-method-parameter name="body" type="object" required=true %}
+{
```
Comment (olegtradecloud) [inline buyer/receive/receive-order-response-api.md:134]: @marcmatt  Do we need to specify here other filters like `companyId`, `contact` etc?
Also json is not properly formatted in the gitbook

image

File: SUMMARY.md [hunk 2]
```diff
@@ -39,8 +39,9 @@
     * [Choose attach document API](buyer/issue/attach-document-api.md)
     * [No delivery expected](buyer/issue/no-delivery-expected.md)
   * [Update an existing order](buyer/update.md)
-  * [Receive an order response](buyer/webhook/README.md)
-    * [Download a document attached to an order response](buyer/webhook/download-document.md)
+  * [Receive an order response](buyer/receive/README.md)
+    * [Choose webhook or polling API](buyer/receive/receive-order-response-api.md)
```
Comment (olegtradecloud) [inline SUMMARY.md:43]: Why do we have both of the `Choose webhook or polling API` pages? Is it possible to extract comon part in the single page ?

**Review by olegtradecloud — CHANGES_REQUESTED** [review]

File: SUMMARY.md (hunk 2 above)
Comment (marcmatt) [reply SUMMARY.md]: @olegtradecloud Good idea, I moved it to api/webhook-vs-polling.md

File: buyer/receive/receive-order-response-api.md (hunk 1 above)
Comment (marcmatt) [reply buyer/receive/receive-order-response-api.md]: Other fields are not relevant for polling.
I tried to use markdown inside the `api-method-parameter` but that is not allowed
image
I cannot find documentation about this.

Comment (olegtradecloud) [reply buyer/receive/receive-order-response-api.md]: Ok, let's leave as is then

File: SUMMARY.md (hunk 2 above)
Comment (olegtradecloud) [reply SUMMARY.md]: Thanks

**Review by olegtradecloud — APPROVED** [review]
Looks very nice!

**Pull request merged**